### ISSUE TITLE
Feature Request: Show all railway tracks when zoomed out further

### ISSUE THREAD

**Issue by tmanok** [OpenRailwayMap/OpenRailwayMap#760]
Hi OpenRailwayMap community!

I absolutely love OpenRailwayMap! Having traveled most of British Columbia on multiple ocassions, I've always wanted to document the existing, abandoned, and incomplete railways that exist. This project has been made much easier with the help of OpenRailwayMap, but because of the size of my province and the vast gaps between railways, I have to zoom in plainfully close to see the majority of northern tracks which are abandoned. This makes understanding the rails and their position in relation to other maps and projects more difficult and time consuming. 

It would be marvellous if there was simply a button under the leaflet controls or under the map style controls areas that allows the user to set minimum zoom visibility for rails. This would add a lot of accessibility to ORM. Optionally, if there is a way to "hack" the local web code to reveal all rails no matter the zoom level, I could happily get along with that. 
Thank you very much!

Tmanok

**Comment by DerDakon** [comment]
Yes, we should start rendering them at zoom 10 or 11, currently they are only visible from 13. @Nakaner, opinions?

**Comment by DerDakon** [comment]
In mapcss we had them up to z9.
